.github/workflows/python-package.yml

```diff
@@ -126,6 +126,7 @@ jobs:
     - name: Install coverage dependency
       if: matrix.python-version == env.GH_COV_PY && matrix.os == env.GH_COV_OS && github.actor != env.GH_DEPENDABOT
       run: |
+        echo 'OTIO_CXX_DEBUG_BUILD=1' >> $GITHUB_ENV
```

Comment:
This would run a debug build only for the coverage cell in the matrix?

Reply:
Exactly. I was under the impression it wouldn't impact much. Is that a good assumption?

Reply:
I read that env var as SGI THUB, and puzzled for a second on what a THUB was

I guess OTIO_CXX_DEBUG_BUILD won't affect the choice of Microsoft CRT? A MSVC build uses the debug CRT, which isn't compatible with non-debug Python distros, which is why I ask

Reply:
It will affect the choice of CRT, but the built DLLs will only be us d by the tests. The wheels will continue to be built in Release mode.

Reply:
ok, sounds good